### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by maolson-msft**

### CODE REVIEW

**Review by goelvidhi — APPROVED** [review]

File: draft-eggert-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -147,10 +147,10 @@ informative:
 CUBIC is an extension to the traditional TCP standards. It differs
 from the traditional TCP standards only in the congestion control
 algorithm on the sender side. In particular, it uses a cubic function
-instead of a linear window increase function of the traditional TCP
+instead of the linear window increase function of the traditional TCP
 standards to improve scalability and stability under fast and
 long-distance networks. CUBIC has been adopted as the default TCP
-congestion control algorithm by Linux, Windows, and Apple stacks.
+congestion control algorithm by the Linux, Windows, and Apple stacks.
```
Comment (goelvidhi) [inline draft-eggert-tcpm-rfc8312bis.md:153]: could you remove the comma after Windows?

File: draft-eggert-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ designed to be less aggressive and fairer to AIMD TCP in bandwidth
 usage than BIC-TCP while maintaining the strengths of BIC-TCP such as
 stability, window scalability, and round-trip time (RTT) fairness.
 CUBIC has been adopted as the default TCP congestion control algorithm
-in Linux, Windows, and Apple stacks, and has been used and deployed
+in the Linux, Windows, and Apple stacks, and has been used and deployed
```
Comment (goelvidhi) [inline draft-eggert-tcpm-rfc8312bis.md:206]: same here

File: draft-eggert-tcpm-rfc8312bis.md (hunk 1 above)
Comment (maolson-msft) [reply draft-eggert-tcpm-rfc8312bis.md:153]: This is valid usage, i.e. an "Oxford Comma." I prefer to use the Oxford Comma but I don't feel strongly about it; let me know if you still want this change.

Comment (larseggert) [reply draft-eggert-tcpm-rfc8312bis.md:153]: What do we do elsewhere? We should be consistent.

Comment (goelvidhi) [reply draft-eggert-tcpm-rfc8312bis.md:153]: I did some quick search, and we are using the "Oxford Comma."

**Review by larseggert — APPROVED** [review]

**Pull request merged**